Assets/Scripts/RobotControllerScript.cs

```diff
@@ -0,0 +1,201 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+public class RobotControllerScript : MonoBehaviour
+{
+
+    public float maxSpeed = 10f;
+    public float maxAcceleration = 10f;
+    public float moveForce = 365f;
+    bool facingRight = true;
+
+    private Rigidbody2D rb2d;
+    Animator anim;
+
+    [SerializeField]
+    private PhysicsMaterial2D RunningPhysicsMaterial, StoppingPhysicsMaterial;
+
+    [SerializeField]
+    private Collider2D playerGroundCollider;
+
+    [HideInInspector] public bool jump = false;
+
+    private bool grounded = false;
+    private bool doubleJump = false;
+    public Transform groundCheck;
+    float groundRadius = 0.2f;
+    public LayerMask WhatIsGround;
+    public float jumpForce = 700;
+    public float jumpVelocity = 7;
+
+    private int count;
+    public Text countText;
```

Comment:
You seem to have a lot of public variables and objects which I believe could be serialized fields or reworked into properties which use the get; set; code. So many public variables really add clutter and allow access of variables to other classes that it should not.